examples/common/python/Makefile

```diff
@@ -36,10 +31,10 @@ build :
 
 install:
 	@echo INSTALLING WHEEL FILE =================
-	pip3 install $(WHEEL_FILE)
+	pip3 install dist/tcf_examples_common*.whl
```

Comment:
Better to use WHEEL_FILE variable.  PR https://github.com/hyperledger/avalon/pull/199 might remove modules outside examples/common/python. check with @Ram-srini

Reply:
ok